homeassistant/components/sensor/juhe_stock.py

```diff
@@ -0,0 +1,165 @@
+"""
+Support for monitoring chinese stock market information provided by Juhe.
+
+Stock Market API of Juhe: https://www.juhe.cn/docs/api/id/21
+
+Example configuration.yaml entry
+sensor:
+  - platform: juhe_stock
+    key: xxxxxxxxxxxxxxxxxxxx
+    symbols:
+      - sz000002
+      - sh600600
+      - sh600000
+
+key(Required): Key from Stock Market API of Juhe.
+symbols(Optional): List of stock market symbols for given companies.
+    Default value is sz000002 (万科A).
+"""
+import logging
+import json
+import asyncio
+
+import voluptuous as vol
+
+import http.client
+
+import homeassistant.helpers.config_validation as cv
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+from homeassistant.const import ATTR_ATTRIBUTION
+from homeassistant.helpers.entity import Entity
+
+_LOGGER = logging.getLogger(__name__)
+
+ATTR_OPEN = 'open'
+ATTR_PREV_CLOSE = 'prev_close'
+ATTR_HIGH = 'high'
+ATTR_LOW = 'low'
+ATTR_NAME = 'friendly_name'
+
+CONF_ATTRIBUTION = "Chinese stock market information provided by Juhe"
+CONF_SYMBOLS = 'symbols'
+CONF_KEY = 'key'
+
+DEFAULT_SYMBOL = 'sz000002'
+
+ICON = 'mdi:currency-cny'
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
+    vol.Optional(CONF_SYMBOLS, default=[DEFAULT_SYMBOL]):
+        vol.All(cv.ensure_list, [cv.string]),
+    vol.Required(CONF_KEY): cv.string,

```

Comment:
Please consider to allow the user to use `CONF_NAME` to set an optional prefix/postfix instead of only using the symbol name.